client/reader/stats.js

```diff
@@ -96,6 +96,11 @@ export function recordTrack( eventName, eventProperties ) {
 		eventProperties = Object.assign( { subscription_count: subCount }, eventProperties );
 	}
 
+	if ( location === 'topic_page' ) {
+		const tag = window.location.pathname.split( '/tag/' ).pop();
```

Comment:
If you use some special characters in the tag name (like 未分類), you'll end up with the URL-encoded tag here - worth seeing how it appears when the event is recorded. It might be worth popping a decodeURIComponent into the mix:

<img width="380" alt="screen shot 2017-04-25 at 19 00 16" src="https://cloud.githubusercontent.com/assets/17325/25400210/cbd72ce8-29e9-11e7-90db-bb93deb71aa6.png">